src/main/templates/partials/header.hbs

```diff
@@ -0,0 +1,42 @@
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="mobile.css">
+  <title>ISDCF {{listTitle}}</title>
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" crossorigin="anonymous"></script>
+  <script type="application/javascript" src="features.js"></script>
+</head>
+
+<body>
+  <nav class="navbar navbar-expand-md navbar-light fixed-top bg-light d-print-none d-flex">
+    <a class="navbar-brand" href="#">ISDCF {{listTitle}}</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item">
+          <a id="json-feed" target="_blank" class="nav-link" href="https://registry.isdcf.com/{{listType}}">JSON Feed</a>
+        </li>
+        <li class="nav-item">
+          <a id="pdf-download" target="_blank" class="nav-link" title="Download as PDF" href="{{pdf_path}}">Download PDF</a>
+        </li>
+      </ul>
+         <form id="search-form" class="form-inline">
+          <div class="input-group mb-2">
+            <input id="search" class="form-control" type="text" placeholder="Search {{listTitle}}">
+            <div class="input-group-append">
+              <input type="button" class="btn btn-secondary clear-filter" value="x"/>
```

Comment:
Replace `x` with `🗙`

Reply:
Chatted with Mike via the phone, going to slightly re-work the search a bit, this might go away altogether. I'll get something Monday or before.

Reply:
@palemieux that character doesn't render for me. Ballot ✗ is nice as well. It must form right angles IMHO :)

@SteveLLamb please check that the upper content is clipped on narrow screens:

<img width="504" alt="Screen Shot 2020-07-24 at 6 46 19 PM" src="https://user-images.githubusercontent.com/11240184/88446522-6be82b00-cddf-11ea-9e43-4fa1f89384d6.png">

Reply:
Upper content only gets "clipped" when menu is not collapsed. Since this changes dependent on device and screen size, it doesn't allow moving the content on the page down based on the length of the menu.

Reply:
@radford-for-smpte This is now fixed - swapped out "fixed-top" to "sticky-top" in navbar. This allows for the fixed top in browsers that support (all current versions of IE, Chrome, Safari, Firefox, web and mobile), and renders as static top in those that don't support. Print out work much better as well, and no further overlaps of content, even with the collapsable menu as above.